src/Pokedex.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Header from './componets/Header/index';
+
+function App() {
+  return (
+    <div className="App">
+      <Header state="active" />
```

Comment:
you should not send if is active or not, something on the header is always active, you should check who is active